src/express-lane-auction/ExpressLaneAuction.sol

```diff
@@ -431,6 +433,22 @@ contract ExpressLaneAuction is
         );
     }
 
+    /// @inheritdoc IExpressLaneAuction
+    function setTransferor(Transferor calldata transferor) external {
+        // if a transferor has already been set, it may be fixed until a future round
+        Transferor storage currentTransferor = transferorOf[msg.sender];
+        if (
+            currentTransferor.addr != address(0) &&
+            currentTransferor.fixedUntilRound > roundTimingInfo.currentRound()
```

Comment:
maybe its better to fix to a timestamp (and rounding up 1 round) instead? otherwise changing round time might make the fix duration change

Reply:
I think that's fine - round duration changes might also warrant a change of fix time. Also changing to timestamp might mean that you cant guarantee that the next round will have a fixed transferor